src/devices/Sgp30/Sgp30.cs

```diff
@@ -0,0 +1,279 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using System.Threading;
+using Iot.Device.Common;
+using UnitsNet;
+
+namespace Iot.Device.Sgp30
+{
+    /// <summary>
+    /// Device binding for Sensorion SGP30 TVOC + eCO2 Gas Sensor.
+    /// </summary>
+    public class Sgp30 : IDisposable
+    {
+        // SGP30 Air Quality Sensor I2C Commands
+        // NOTE: The 'Measure Test' command is used for production verification only and is not included here
+        private const ushort SGP30_INITIALISE_AIR_QUALITY = 0x2003;
```

Comment:
Are thos constants you can find in the docs? If yes, fine to leave them in upper case, if not, better to have them with the normal naming convention we are using so `Sgp30InitialiseAirQuality` for example.
Also, you may want o tranform this as an internal enum called Register, see the pattern in other bindings.